tank/model_utils_tf.py

```diff
@@ -57,7 +95,7 @@ def get_TFhf_model(name):
         text,
         padding="max_length",
         truncation=True,
-        max_length=MAX_SEQUENCE_LENGTH,
+        max_length=128,
```

Comment:
Why is this changed when MAX_SEQUENCE_LENGTH still is 128? Ideally don't use numbers when you can use a named variable

Reply:
This was a temporary fix to a problem that I have fixed in model_utils_tf. See new commit.

Reply:
You can still set the #define to 128. In general numbers in code should be #defines